packages/myst-cli/src/process/notebook.ts

```diff
@@ -109,41 +103,41 @@ export async function processNotebook(
         return acc.concat(blockParent(cell, cellMdast.children));
       }
       if (cell.cell_type === CELL_TYPES.raw) {
-        const cellContent = `\`\`\`\n${ensureString(cell.source)}\n\`\`\``;
-        const cellMdast = parseMyst(session, cellContent, file);
-        return acc.concat(blockParent(cell, cellMdast.children));
+        const raw: Code = {
+          type: 'code',
+          lang: '',
+          value: ensureString(cell.source),
+        };
+        return acc.concat(blockParent(cell, [raw]));
       }
       if (cell.cell_type === CELL_TYPES.code) {
-        const cellCodeContent = `\`\`\`{code-cell} ${language}\n${ensureString(
-          cell.source,
-        )}\n\`\`\``;
-        const cellCodeMdast = parseMyst(session, cellCodeContent, file);
-        const { myst, id } = createOutputDirective();
+        const code: Code = {
+          type: 'code',
+          lang: language as string | undefined,
+          executable: true,
+          value: ensureString(cell.source),
+        };
+
+        const output: { type: 'output'; id: string; data: MinifiedOutput[] } = {
+          type: 'output',
+          id: nanoid(),
+          data: [],
+        };
+
         if (cell.outputs && (cell.outputs as IOutput[]).length > 0) {
           const minified: MinifiedOutput[] = await minifyCellOutput(
             cell.outputs as IOutput[],
             cache.$outputs,
             { computeHash, maxCharacters: opts?.minifyMaxCharacters },
           );
-          outputMap[id] = minified;
-        } else {
-          outputMap[id] = [];
+          output.data = minified;
```

Comment:
seems like this could be much simplier, like this... not clear on the need for the outputMap, unless cells were sharing outputs somehow? but that's not possible in a notebook?? 🤔